dashboard/prepare-dashboard-data.py

```diff
@@ -300,6 +301,14 @@ def count_dups(hvr_fname):
         sample_metadata[sample]["reads"] = \
             project_sample_reads[project][sample]
 
+        rc_fname = "ribocounts/%s.ribocounts.txt" % sample
+        try:
+            with open(rc_fname, 'r') as file:
+                ribocount = int(file.read().strip())
+            sample_metadata[sample]["ribocounts"] = ribocount
+        except FileNotFoundError:
+            pass
+
```

Comment:
It looks like you pulled in the changes from https://github.com/naobservatory/mgs-pipeline/pull/21 the wrong way, and it thinks those are new on your branch?

Reply:
I think this happened because I used the same branch for #20, which predates #21. I'm planning to squash and merge, which I think will render that irrelevant.

Reply:
Sounds good!